msteams-platform/sbs-botbuilder-taskmodule.yml

```diff
@@ -153,10 +153,10 @@ items:
 
     1. Select your bot from **Recent resources**.
 
-    1. Select **Channels** in the left pane. 
-
-    1. Select **Microsoft Teams** <img src="~/assets/images/bots/teamsicon.png" alt="Teams icon" width="20"/>.
```

Comment:
Wow! I did not know that inline HTML was used for images in our docs. Thank you for fixing this instance. I have created an ADO to update all such instances. 
CC @v-habibaumme @surbhigupta12